core/src/main/scala/knobs/AWS.scala

```diff
@@ -36,59 +37,67 @@ object aws {
 
   private val defaultFormating: String => String = x => "\"" +x+ "\""
 
+  private def or[A](x: IO[A], y: => IO[A]): IO[A] =
```

Comment:
Why is `y` lazy?

Reply:
Wanted to avoid evaluating it in the case of a `Right`, more habit than anything. I can get rid of it if you prefer.

Reply:
If the IO is expensive to construct, in addition to being dangerous to evaluate, then it still makes sense.  I don't have a moral objection.  It just looks weird on an IO.